pkg/scheduler/factory.go

```diff
@@ -463,7 +463,7 @@ func MakeDefaultErrorFunc(client clientset.Interface, podLister corelisters.PodL
 				}
 			}
 		} else {
-			klog.Errorf("Error scheduling %v/%v: %v; retrying", pod.Namespace, pod.Name, err)
+			klog.ErrorS(err, "Error scheduling; retrying", "pod", klog.KObj(pod))
```

Comment:
```suggestion
			klog.ErrorS(err, "Unexpected error", "pod", klog.KObj(pod))
```

Reply:
@serathius log lines 448 and 450 should be extremely common for managed k8s (AKS, GKE and EKS), I think we should convert them as well.

Reply:
I think it's ok to migrate additional lines assuming that it's reasonably fast. @tanjunchen What do you think?
If not I would not block on it and add migrating those lines to [Structured logging tracker](https://github.com/kubernetes/enhancements/issues/1602) as we have more contributors then tasks.

Reply:
@serathius  agree. it is good.

Reply:
updated